Makefile

```diff
@@ -44,6 +46,9 @@ start-dockerdemo:
 clean-dockerdemo:
 	scripts/dockerhost/cleanup-dockerhosts
 
+ssh:
```

Comment:
Ah, I didn't mean to setup anything extra than what exists. I was wondering if something like below will ensure that vagrant environment is up before trying to ssh:

```
ssh:
   make demo || true
   vagrant ssh netplugin-node1
```
But this works as well I guess sin ssh will fail if environment is not up already.